### PR TITLE
Updated paths to accomodate the move of many scripts from ./scripts/ to ./bin/

### PR DESCRIPTION
**Pull request by ibennetch**
Related to phpmyadmin/phpmyadmin#18798
Fixes https://github.com/phpmyadmin/website/issues/165

### CODE REVIEW

**Comment by ibennetch** [comment]
I think I got all of the references in the scripts repository here.

**Review by williamdes — APPROVED** [review]

File: website/daily-snapshots [hunk 1]
```diff
@@ -21,7 +21,11 @@ git checkout QA_5_2
 git reset --hard
 # Force an update
 git pull --ff-only
-./scripts/create-release.sh --daily 5.2+snapshot QA_5_2
+if [ -f ./scripts/create-release.sh ]; then
+  ./scripts/create-release.sh --daily 5.2+snapshot QA_5_2
+else
+  ./bin/internal/create-release.sh --daily 5.2+snapshot QA_5_2
```
Comment (MauricioFauth) [inline website/daily-snapshots:27]: I'm not sure if this is necessary, as the QA_5_2 branch is checked out a few lines earlier.

**Review by MauricioFauth — APPROVED** [review]

File: website/daily-snapshots (hunk 1 above)
Comment (ibennetch) [reply website/daily-snapshots]: Thanks, good point. I reverted this and added a comment.

**Comment by ibennetch** [comment]
I'm happy with this now; if there are no further comments I'll merge later today.

**Comment by ibennetch** [comment]
I made some adjustment to check for the folder outside of the if loop in pma-demo-update and reverted the part @MauricioFauth mentioned will always been on QA_5_2 (plus added a note for when we migrate to QA_6_0)

I'm happy with this now; if there are no further comments I'll merge later today.

**Review by MauricioFauth — APPROVED** [review]

**Review by williamdes — APPROVED** [review]

**Pull request merged**